client-v2/src/components/CAPI/SearchQuery.js

```diff
@@ -30,6 +30,8 @@ class SearchQuery extends React.Component<
     params: {}
   };
 
+  state = {};
+
```

Comment:
Why in this PR?

Reply:
Because I was getting some linting error and I got annoyed and just did it 😞